readers/hdfs/src/main/java/com/criteo/hadoop/garmadon/hdfs/writer/PartitionedWriter.java

```diff
@@ -269,7 +269,7 @@ private boolean tryExpireConsumer(ExpiringConsumer<MESSAGE_KIND> consumer) {
                 consumer.close();
                 return true;
             } catch (IOException e) {
-                LOGGER.error("Couldn't close writer for {} ({}/{})", eventName, retry, maxAttempts, e);
+                LOGGER.error("Couldn't close writer for {} ({}/{}): {}", eventName, retry, maxAttempts, e.getMessage());
```

Comment:
use String.format and pass the exception as second parameter to LOGGER.error to keep stack trace